detector/ScienceEnergyChannels_01023023.csv

```diff
@@ -0,0 +1,57 @@
+ENERGY BINNING,,,,,,,01.02.23
+,,,,,,,
+Energy binning with masks reset after January 2023 fine energy test but ELUT unchanged.,,,,,,,
+,,,,,,,
+Criteria,,,,,,,
+,Increasing bin width between 4 keV and 150 keV,,,,Include LLD and ULD channels < 4kev and > 150keV,,
+,"Fine (< 1kev) resolution through microflare regime, 4-11 keV",,,,"Bracket Ba-133 lines at 31, 35 and 81 keV",,
+,Bin centres at 6.7 and 8.0 keV (Fe and Fe/Ni line complexes),,,,,,
+,Sufficient coverage over full range,,,,,,
+,"Match current QL boundaries at 4, 10, 15, 25, 50, and 84 keV",,,,,,
```

Comment:
Sorry, my mistake but this line shouldn't be here as this configuration does not match the standard QL boundaries.

Reply:
So I should just remove this `"Match current QL boundaries at 4, 10, 15, 25, 50, and 84 keV"`

Reply:
[ScienceEnergyChannels_01022023.csv](https://github.com/i4Ds/STIX-CONF/files/10697389/ScienceEnergyChannels_01022023.csv)
Attached is an updated version with one of the additional criteria from column 6 moved to column 2. I also noticed there was a typo in the file name!

Reply:
What should that file be called, I guess it is the creation date the other one looks to be yyyymmdd and the new one ddmmyyyy?